core/src/trezor/wire/context.py

```diff
@@ -79,6 +79,16 @@ async def call_any(
     return await CURRENT_CONTEXT.read(expected_wire_types)
 
 
+async def write(msg: protobuf.MessageType) -> None:
```

Comment:
this should probably be left out of this PR and done a bit differently in #4919

Reply:
Removed in https://github.com/trezor/trezor-firmware/commit/0b1ea106fca6cf68597987863a39f3eb8eb7ef00.